UIKit+Combine/UICollectionView+Combine.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  UICollectionView+Combine.swift
+//  CombineCocoa
+//
+//  Created by Joan Disho on 05.04.20.
+//  Copyright © 2020 Joan Disho. All rights reserved.
+
+import Foundation
+import UIKit
+import Combine
+
+private class CollectionViewDelegateProxy: DelegateProxy, UICollectionViewDelegate, DelegateProxyType {
+    func setDelegate(to object: UICollectionView) {
+        object.delegate = self
+    }
+}
+
+extension UICollectionView {
+    private var delegateProxy: CollectionViewDelegateProxy {
```

Comment:
ACL order should be public at the top, internal and private at the bottom in extensions, prefferably (e.g. `public extension` on top, then `// MARK: - Private Helpers` and then `extension / private extension` accordingly)

This is true for all other files as well